r/tests/testthat/test-dplyr-funcs-datetime.R

```diff
@@ -46,6 +46,46 @@ test_df <- tibble::tibble(
   integer = 1:2
 )
 
+# a simplified test case using UTC timezone only
+test_df_v2 <- tibble::tibble(
+  datetime = c(as.POSIXct("2017-01-01 00:00:11.3456789", tz = "UTC"), NA) + 1,
```

Comment:
I wonder if (1) we want to test non-UTC times too? Would that change anything? and (2) if we do use a timezone that's not UTC, we should also have times that would be on either side of midnight on different days _in_ UTC for cases where we rollup to days, we want to make sure that does the right thing.

Reply:
I think we should. I was having difficulties handling timezones when I started the PR so I sidestepped this, but I think it's worth revisiting this before we merge it

Reply:
🤞